src/batch/monitor.rst

```diff
@@ -31,13 +37,15 @@ The ``scontrol`` command can be used to show detailed job information.
 
    [testuser@fe-ac-02 ~]$ scontrol show jobid -dd <jobid>
```

Comment:
If you remove <> everywhere else, you should do the same for `<jobid>` here

Reply:
OK